web/src/components/vis/VisTile.vue

```diff
@@ -48,6 +49,9 @@ export default {
     setScaleIndex(value) {
       this.scaleIndex = Math.max(Math.min(Number.parseInt(value, 10), this.scales.length - 1), 0);
     },
+    hasHelp() {
+      return this.$slots.help != null;
```

Comment:
`this.$slots.help` cannot be null, so this check relies on coercion without appearing to do so.  Either `this.$slots.help !== undefined` (no coercion) or `!!this.$slots.help` (obvious coercion) would probably be better.

Reply:
`this.$slots.help != null` checks for both `!== null` and `!== undefined` at the same time. That's why I just used `!=` and not `!==`

Reply:
note: and `!= null` is the only time a good linter will allow not to use `!==` for comparison